packages/frontend/src/pages/timeline.vue

```diff
@@ -236,6 +239,19 @@ function closeTutorial(): void {
 	defaultStore.set('timelineTutorials', before);
 }
 
+function switchTlIfNeeded() {
+	if (isBasicTimeline(src.value) && !availableBasicTimelines().includes(src.value)) {
```

Comment:
[Non Blocker]

```suggestion
	if (isBasicTimeline(src.value) && !isAvailableBasicTimeline(src.value)) {
```

importを追加する必要があります

Reply:
あったのか